deploy/kubernetes/filebeat-kubernetes.yaml

```diff
@@ -89,6 +89,8 @@ spec:
           value:
         securityContext:
           runAsUser: 0
+          # If using Red Hat OpenShift uncomment this:
+          #privileged: true
```

Comment:
this file is autogenerated, that's why tests are failing. You will need to update https://github.com/elastic/beats/blob/master/deploy/kubernetes/filebeat/filebeat-daemonset.yaml and run `make`

Reply:
@exekias Can we add comments to the generated file so that folks (like me) will know this in the future?

Reply:
@exekias That's a good point, and I agree. It would be confusing to users, so let's not add the comment. That's why we have the tests, right.